tests/integration/utils.py

```diff
@@ -280,7 +283,13 @@ def generate_cfgs(unified_testing=False, dry_run=False):
     )
     expansions = get_expansions()
     if unified_testing:
+        expansions["diags_environment_commands"] = expansions[
```

Comment:
Might be useful to wrap this in a `if expansions["diags_environment_commands"] == "":`, so it doesn't overwrite an explicitly set `diags_environment_commands`. Currently testing with Unified only, so fine for now.

Reply:
Addressed at https://github.com/E3SM-Project/zppy/pull/676